### PR TITLE
test(plugin-import): don't test core

### PR DESCRIPTION
**Pull request by flipactual**
**What**: stop testing `@emdaer/core` for `@emdaer/plugin-import`

**Why**: because `@emdaer/plugin-import` is testing `@emdaer/core`

**How**: You get it

**Checklist**:

- [ ] Documentation N/A
- [x] Tests
- [x] Ready to be merged 
- [x] Added myself to AUTHORS

### CODE REVIEW

**Review by infiniteluke — DISMISSED** [review]
Looks like things are broken. I’m seeing `false` for every readme

**Comment by flipactual** [comment]
Bleh, precommit woes

**Comment by flipactual** [comment]
Updated

File: packages/plugin-import/src/index.test.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
 jest.mock('fs-extra');
 
+jest.mock('@emdaer/core', () => async () => 'Hello, World!', {
```
Comment (infiniteluke) [inline packages/plugin-import/src/index.test.js]: There shouldn't be a need for a mock implementation. You can just check if it was called like here:
https://github.com/emdaer/emdaer/pull/56/files#diff-9cfed617b4f3c8a2119339cb62ec388eR90

**Review by infiniteluke — APPROVED** [review]
One nit, but nbd

File: packages/plugin-import/src/index.test.js (hunk 1 above)
Comment (infiniteluke) [reply packages/plugin-import/src/index.test.js]: You'll have to require it in after the `jest.mock` and so that you have something to pass to `expect`

Comment (flipactual) [reply packages/plugin-import/src/index.test.js]: Forgot that `@emdaer/core` was in this one... and good call on not needing to mock the implementation

**Pull request merged**